ledger/query.py

```diff
@@ -0,0 +1,114 @@
+"""Utilities for GitHub GraphQL querying."""
+from functools import lru_cache
+from pathlib import Path
+from typing import Any
+
+
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport
+from graphql import DocumentNode
+
+from ledger.config import GitHubConfig
+
+
+QUERIES = []
+
+# Select your transport with a defined url endpoint
+transport = AIOHTTPTransport(
+  url="https://api.github.com/graphql",
+  headers={
+    "Authorization": f"bearer {GitHubConfig.token}"
+  }
+)
+
+# Create a GraphQL client using the defined transport
+client = Client(
+  transport=transport,
+  fetch_schema_from_transport=True
+)
+
+
+@lru_cache()
+def load_query(name: str) -> DocumentNode:
+    """Load a query from the queries folder."""
+    query_file = Path(__file__).parent / f"queries/{name}.gql"
```

Comment:
It may be necessary to add a base directory path in the future. What do you think?

Reply:
I thought about it for approximately 2 seconds and I take it back. Everything's good.